app/config/locale/translations/fa.countries.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+return [
+    'AF' => 'افقانستان',
+    'AO' => 'آنگولا',
+    'AL' => 'آلبانی',
+    'AD' => 'آندورا',
+    'AE' => 'امارات متحده عربی',
+    'AR' => 'آرژانتین',
+    'AM' => 'ارمنستان',
+    'AG' => 'آنتیگوا و باربودا',
+    'AU' => 'استرالیا',
+    'AT' => 'اتریش',
+    'AZ' => 'آذربایجان',
+    'BI' => 'بوروندی',
+    'BE' => 'بلژیک',
+    'BJ' => 'بنین',
+    'BF' => 'بورکینافاسو',
+    'BD' => 'بنگلادش',
+    'BG' => 'بلغارستان',
+    'BH' => 'بحرین',
+    'BS' => 'باهاما',
+    'BA' => 'بوسنی و هرزگوین',
+    'BY' => 'بلاروس',
+    'BZ' => 'بلیز',
+    'BO' => 'بولیوی',
+    'BR' => 'برزیل',
+    'BB' => 'باربادوس',
+    'BN' => 'برونئی',
+    'BT' => 'بوتان',
+    'BW' => 'بوتسوانا',
+    'CF' => 'جمهوری آفریقای مرکزی',
+    'CA' => 'کانادا',
+    'CH' => 'سوئیس',
+    'CL' => 'شیلی',
+    'CN' => ']dk',
```

Comment:
Hey @irajtaghlidi please retranslate this

Reply:
Thank you @spielers for your review, I fixed the 'CN' typo one.